### ISSUE TITLE
quantitative model request

### ISSUE THREAD

**Issue by callmehanyu** [curiosity-ai/florence2-sharp#2]
Can you provide a quantized version using q4f16? I tried to replace it, but the code gave an error:

`Unhandled exception. Microsoft.ML.OnnxRuntime.OnnxRuntimeException: [ErrorCode:Fail] Load model from ./models/decoder_model_merged_q4f16.onnx failed:/Users/runner/work/1/s/onnxruntime/core/graph/graph.cc:1421 void onnxruntime::Graph::InitializeStateFromModelFileGraphProto() This is an invalid model. Subgraph output (logits) is an outer scope value being returned directly. Please update the model to add an Identity node between the outer scope value and the subgraph output.`
